src-docs/src/assets/version-selector.js

```diff
@@ -0,0 +1,230 @@
+
+const PREFIX = 'v';
+
+const tpl = `
+    <style>
+    :host {
+        display: inline-block;
+        position: relative;
+        box-sizing: border-box;
+        font-size: 12px;
+        user-select: none;
+        margin: 3px;
+        text-align: left;
+    }
+    
+    * {
+        box-sizing: border-box;
+    }
+    
+    #root {
+        text-decoration: none;
+        color: #FFFFFF;
+        // background-color: #00509c;
```

Comment:
Please remove commented code if you don't need

Reply:
done